mobify/platform-scaffold
# Fix carousel bug

Fix carousel bug

 **JIRA**: N/A
 **Linked PRs**: MERGE THIS PR first: https://github.com/mobify/platform-scaffold/pull/830

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions PDP](http://localhost:3000/potions/eye-of-newt.html)
- Make sure that arrows show all the time
